src/Common/src/TypeSystem/IL/ILImporter.cs

```diff
@@ -403,7 +403,7 @@ private void ImportBasicBlock(BasicBlock basicBlock)
                         break;
                     case ILOpcode.jmp:
```

Comment:
We really ought to have an assert that tells us that we aren't processing instructions beyond the end of the current basic block.